mongo/mongo.go

```diff
@@ -582,20 +582,44 @@ func ensureServer(args EnsureServerParams, mongoKernelTweaks map[string]string)
 		return mongodVersion, nil
 	}
 
-	running, err := svc.Running()
+	// Installed tells us if there exists a service of the right name.
+	installed, err := svc.Installed()
 	if err != nil {
 		return zeroVersion, errors.Trace(err)
 	}
-	if running {
-		return mongodVersion, nil
+	if installed {
+		// Exists does a check against the contents of the service config file.
```

Comment:
It might be easier to read as: 'Exists()' rather than just 'Exists'.

Reply:
Seems reasonable to me. Updated.